### PR TITLE
CDAP-4357 Fix the expected count of pending queue events

### PR DESCRIPTION
**Pull request by gokulavasan**
JIRA : https://issues.cask.co/browse/CDAP-4357

Build : http://builds.cask.co/browse/CDAP-RBT568

### CODE REVIEW

**Comment by gokulavasan** [comment]
Build is :green_apple:. @anew Please review when you get a chance. Thanks!

File: cdap-app-fabric/src/test/java/co/cask/cdap/runtime/FlowTest.java [hunk 1]
```diff
@@ -294,14 +294,18 @@ public void testFlowPendingMetric() throws Exception {
       // source emits 4, then forward-one reads 1, hence 3 should be pending
       waitForPending(tagsForSourceToOne, 3, 5000); // wait a little longer as flow needs to start
       waitForPending(tagsForAllToOne, 3, 100); // wait a little longer as flow needs to start
-      // forward-two receives each of the 4 as a sting and an int, but has read only 1
-      // so there should be 3 + 4 = 7 pending, but we don't know which queue has 3 and which has 4
+      // forward-two receives each of the 4 as a sting and an int, but could have read 1 per each queue
```
Comment (sreevatsanraman) [inline cdap-app-fabric/src/test/java/co/cask/cdap/runtime/FlowTest.java]: typo string

**Comment by sreevatsanraman** [comment]
one minor nit. rest LGTM

File: cdap-app-fabric/src/test/java/co/cask/cdap/runtime/FlowTest.java (hunk 1 above)
Comment (anew) [inline cdap-app-fabric/src/test/java/co/cask/cdap/runtime/FlowTest.java]: I like it, it's stingy :)

**Comment by anew** [comment]
LGTM

**Pull request merged**